public/js/safetymaps/modules/incidents/VehicleIncidentsController.js

```diff
@@ -1007,7 +1011,76 @@ VehicleIncidentsController.prototype.geenInzet = function() {
     this.incidentDetailsWindow.hideMultipleFeatureMatches();
 };
 
+VehicleIncidentsController.prototype.saveKladblokChatRow = function (row, incidentnr) {
+    var me = this;
+    if (me.options.editKladblokChatAuthorized && row.length > 0) {
+        $.ajax("api/kladblok/" + incidentnr + ".json", {
+            method: 'POST',
+            data: {
+                vehicle: dbkjs.options.organisation.username,
```

Comment:
If we use the username instead of the voertuignummer (because the voertuignummer setting can be multiple voertuignummers), the server should just fill this in server-side, and don't call it vehicle but just username.